registrations/serializers.py

```diff
@@ -500,6 +500,13 @@ class Meta:
         model = SeatReservationCode
 
 
+class MassEmailSignupGroupsField(serializers.PrimaryKeyRelatedField):
+    def get_queryset(self):
+        registration = self.context["request"].data["registration"]
+
+        return registration.signup_groups.exclude(signups__email=None)
```

Comment:
If a single signup inside a signup group has email as `None` the signup group will be excluded, I guess that is intended?

Reply:
This would have been wrong actually => now changed so that all groups for the registration are included, but later the responsible signups are filtered like this.